GoRogue/MapViews/IMapViewExtensions.cs

```diff
@@ -337,5 +337,25 @@ public static Point RandomPosition<T>(this IMapView<T> mapView, IGenerator? rng
 
             return new Point(rng.Next(mapView.Width), rng.Next(mapView.Height));
         }
+
+        /// <summary>
+        /// Converts this IMapView<T> into an array of T[]
```

Comment:
Can't use < or > in comments (badly formed XML); need to either use the XML escapes for those characters or refs, or simply omit the type since it's implicit in the function signature:
```suggestion
        /// Converts the given map view into an array of T[]
```